### PR TITLE
adding models,repositories

### PR DESCRIPTION
**Pull request by mherman22**
- added models
- added repositories
- changed java version from 17 to 1.8

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	<name>KTMInv</name>
 	<description>Backend for the KTMInv</description>
 	<properties>
-		<java.version>17</java.version>
+		<java.version>1.8</java.version>
```
Comment (mherman22) [inline pom.xml:17]: will change this back probably in the near future.

File: src/main/java/com/mherman22/KTMInv/models/Customer.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
```
Comment (mherman22) [inline src/main/java/com/mherman22/KTMInv/models/Customer.java:12]: used lombok in the model classes in order to reduce on the boilerplate code (generating getters,setters,constructors,etc..)

File: src/main/java/com/mherman22/KTMInv/models/Payment.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	
+	@Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long paymentID;
+	
+	@Column(name = "payment_mode")
+	private String paymentMode;
+	
+	@Column(name = "payment_date")
+	private Date paymentDate;
+
```
Comment (kinghk-debug) [inline src/main/java/com/mherman22/KTMInv/models/Payment.java:30]: remove this line!

**Review by kinghk-debug — CHANGES_REQUESTED** [review]

**Review by kinghk-debug — APPROVED** [review]

**Review by kinghk-debug — APPROVED** [review]
connect mysql to this application as the next step. othewise thx

**Pull request merged**